README.md

```diff
@@ -12,7 +12,11 @@ Ensure the following are installed on your system before proceeding:
 
 This is our deployed website link https://love-me-tender-51qa.onrender.com
 
-The render is on auto-deploy so it should render new changes to the website.
+To get the latest version deployed on render;
+
+1. Go to render dashboard
+2. Click on manual deploy button
+3. Select deploy latest commit
```

Comment:
Thanks for updating this. In the future, please add it in a different commit, since it's unrelated to the other changes

Reply:
Okay next time I will make a different commit iof its different from the task

Reply:
It's probably good practice to do it now. Create a new commit just with this change and remove it from this commit.

Reply:
I have committed it separately